app/gui/controller/double-representation/src/node.rs

```diff
@@ -458,7 +458,7 @@ impl NodeAst {
 
     /// AST of the node's expression. Typically no external user wants to access it directly. Use
     /// [`Self::expression`] instead.
```

Comment:
Docs accurate?

Reply:
Yep, it's still not typical to use it, but in this case we need it.